tests/baselines/reference/override12(target=esnext).js

```diff
@@ -0,0 +1,45 @@
+//// [override12.ts]
+class A {
+    public m1(): number {
+        return 0;
+    }
+
+    public m2(): number {
+        return 0;
+    }
+
+    public m3(): void {}
+}
+
+class B extends A {
+    override m1() {
```

Comment:
🤦‍♂️

Reply:
@DanielRosenwasser ?

Reply:
Holy shit🤦‍♂️

Reply:
@Kingwl @DanielRosenwasser Could you clarify this case? `//// [override12.js]` does not contain `override` keywords.

Reply:
But if `useDefineForClassFields` is on....

[playground](https://www.typescriptlang.org/play?useDefineForClassFields=true&target=99&noImplicitOverride=true&ts=4.4.0-dev.20210526#code/MYGwhgzhAEDC0G8BQ1XTNAvNAjEgvkkqJDACLQCmAHgC6UB2AJjPMmtAPYBulATnwCWTSuizQATASA)